packages/@aws-cdk/aws-stepfunctions-tasks/README.md

```diff
@@ -1111,6 +1111,10 @@ new sfn.StateMachine(this, 'ParentStateMachine', {
 });
 ```
 
+You can set `associateWithParent=true` to utilize
+[Associate Workflow Executions](https://docs.aws.amazon.com/step-functions/latest/dg/concepts-nested-workflows.html#nested-execution-startid),
```

Comment:
give a little more detail about what the benefit is

Reply:
I added this sentence straight from the docs: "This makes it easy to trace executions of your workflows
from starting executions to their started workflow executions." However this seems poorly written to me unless it makes more sense to you

Reply:
I think that's fine...but something better could be "This allows the Step Functions UI to link child executions from parent executions, making it easy to trace execution flow across state machines". Feel free to iterate! I just want some amount of detail about what the end result for the customer is

Reply:
I like that. I assume making the distinction between generic "Stepfunctions" and "Stepfunctions UI" is to show that the customer benefit lies in the console, right?

Reply:
Correct – as far as I know, this is simply for ease of investigation through the console